types/@html-next/vertical-collection.d.ts

```diff
@@ -21,12 +22,32 @@ export interface VerticalCollectionArgs<T> {
 
 export interface VerticalCollectionSignature<T> {
     Element: HTMLElement;
-    Args: { Named: VerticalCollectionArgs<T>; Positional: { items: NativeArray<T> | T[] } };
+    Args: {
+        Named: {
+            items: NativeArray<T> | T[];
```

Comment:
It might make more sense to re-use the VerticalCollectionArgs here instead of redefining everything?

Reply:
Do we need to export VerticalCollectionArgs separately?

Reply:
Could go either way - I think we maybe used those in a project before so we didn't have to redeclare it _in_ the project. But also we can just declare them in the project since this whole thing is suppose to go away someday.